docs/kubernetes-cluster-management/eks.md

```diff
@@ -0,0 +1,31 @@
+# Elastic Kubernetes Service (EKS)
+EKS is a managed Kubernetes service provided by AWS.
+
+## Role Based Access Control (RBAC)
+Role based access control (RBAC) should be set up to ensure developer AWS accounts are able to access the cluster via kubectl and helm.  
+
+Further information can be found within the [AWS documentation](https://docs.aws.amazon.com/eks/latest/userguide/add-user-role.html).
+
+### Checking current user ARN
+To check you are using your correct AWS account run the `aws iam get-user` command.
+
+## Interact with EKS cluster
+All of the below process require this first step to be complete as a prerequisite.
+
+- connect to FFC AWS VPN using 2FA
+- clone private repo containing kubeconfig file for relevant cluster
```

Comment:
I'm not sure what this means - is this a gitlab repo?

Reply:
For each cluster we have a private repo that contains a kubeconfig file.  This file can be used to access a cluster and interact with it.  So the first step is cloning that repo to get the file locally.  For AWS the configuration file will be GitLab, on Azure it will be Azure Repos.

I was in two minds about whether to specifically mention GitLab and didn't as we have private repos in Azure.  But I guess we're talking about EKS and I can't imagine a scenario where we'd have and EKS cluster kubeconfig in an Azure repo.  I'm starting to think it would make sense to mention GitLab by name.  Thoughts?

Reply:
I don't see any problem with making it specific to GitLab: if ever we add an EKS cluster kubeconfig to an Azure repo, we can clarify the instructions then. It'll certainly make it clearer to a newby like me!

Reply:
Making it clearer sounds good to me!  Done, I've also added a bit more explanation about kubeconfig files.